server/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTermsAggregationBuilder.java

```diff
@@ -129,6 +129,20 @@ public SignificantTermsAggregationBuilder(StreamInput in) throws IOException {
         significanceHeuristic = in.readNamedWriteable(SignificanceHeuristic.class);
     }
 
+    protected SignificantTermsAggregationBuilder(SignificantTermsAggregationBuilder clone) {
+        super(clone);
+        this.bucketCountThresholds = clone.bucketCountThresholds;
```

Comment:
It seems that BucketCountThresholds is mutable so I think we need to clone it as well?